src/app/application-form/application-form.component.html

```diff
@@ -559,9 +578,11 @@ <h1>Please fill this application form</h1>
 
             <div>
               <button mat-button matStepperPrevious>Back</button>
-              <button mat-button (click)="stepper.reset()">Reset</button>
               <button mat-button (click)="onPreview()" (click)="openDialog()">
```

Comment:
Should this button submit the form?
If it should - it should have `type="submit"` on it .